selfdrive/car/honda/hondacan.py

```diff
@@ -142,16 +142,24 @@ def create_ui_commands(packer, pcm_speed, hud, car_fingerprint, is_metric, idx,
     'SOLID_LANES': hud.lanes,
     'BEEP': 0,
   }
-  commands.append(packer.make_can_msg('LKAS_HUD', bus_lkas, lkas_hud_values, idx))
 
-  if radar_disabled and car_fingerprint in HONDA_BOSCH:
+  if CP.flags & Flag.BOSCH_EXT_HUD:
+    lkas_hud_values['SET_ME_X48'] = 0x48
+
+  if CP.flags & Flag.BOSCH_EXT_HUD and not CP.openpilotLongitudinalControl:
```

Comment:
@csouers why is `openpilotLongitudinalControl` checked here?

Reply:
The radar passes along the new LKAS_HUD messages to the powertrain bus on 0x33d. If it's disabled, we just send LKAS_HUD like usual.